app/frontend/src/components/designer/FormDisclaimer.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-  <div>
+  <div :class="{ 'dir-rtl': isRTL }">
     <h3 class="mb-5">
       {{ $t('trans.formDisclaimer.disclaimerAndStatement') }}
     </h3>
-    <ul>
+    <ul :class="isRTL ? 'mr-6' : null">
       <li>
-        {{ $t('trans.formDisclaimer.privacyLaw') }}
+        <span>
+          {{ $t('trans.formDisclaimer.privacyLaw') }}
+        </span>
```

Comment:
Do we still need these `span` tags? It would be good to make as few changes as possible. Ideally it would be great if everything change in the "file diff" view of the PR was for a reason.